curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/complete-a-promise-with-resolve-and-reject.english.md

```diff
@@ -0,0 +1,80 @@
+---
+id: 5cdafbc32913098997531680
+title: Complete a Promise with Resolve and Reject
+challengeType: 1
+---
+
+## Description
+<section id='description'>
+A promise has three states: <code>pending</code>, <code>fulfilled</code>, and <code>rejected</code>. The promise you created in the last challenge is forever stuck in the <code>pending</code> state because you did add a way to complete the promise. The <code>resolve</code> and <code>reject</code> parameters given to the promise argument are used to do this. <code>resolve</code> is used when you want your promise to succeed, and <code>reject</code> is used when you want it to fail. These are methods that take an argument, as seen below.
+
+```js
+const myPromise = new Promise((resolve, reject) => {
+	if(true) {
+		resolve("Promise was fulfilled");
+	} else {
+		reject("Promise was rejected");
+	}
+});
+```
+
+The example above uses strings for the argument of these functions, but it can really be anything. Often, it might be an object that you would use data from to put on your website or elsewhere.
+</section>
+
+## Instructions
+<section id='instructions'>
+Use the <code>resolve</code> method if the condition given is true to fulfill the promise. Pass it the string: <code>We got the data.</code> as its argument. Use the <code>reject</code> method if condition is not true and pass it the string: <code>Data not received.</code>.
```

Comment:
```suggestion
Use the <code>resolve</code> method if the condition given is true to fulfill the promise. Pass it the string: <code>We got the data</code> as its argument. Use the <code>reject</code> method if condition is false and pass it the string: <code>Data not received</code>.
```
I think we can get rid of the `.`, I'm sure many people would get errors due to this fullstop